src/pages/gift/generate/PresentGift.js

```diff
@@ -2,31 +2,34 @@ import { Row, Col, Card } from 'react-bootstrap';
 import CardHeader from '../../../components/CardHeader';
 import { useSubstrate, utils } from '../../../substrate-lib';
 import config from '../../../config';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
 
 export default function PresentGift ({ gift, removeGiftHandler }) {
   const { email, amount, secret } = gift;
   const { giftTheme, chainInfo } = useSubstrate();
   const amountStr = amount && utils.formatBalance(amount, chainInfo?.token);
   const mailSubject = `Someone has sent you ${giftTheme?.content}`;
   const claimUrl = config.CLAIM_URL;
-  const mailBody = `
-  Hey! \n 
+
+  const giftMessage = `
+  Hey! \n
```

Comment:
In the copied clipboard text it takes the two spaces indentation and the first line break, so the formatting ends up looking like:
```


  Hey!

  I'm sending you 100 DOTs as a gift! You can go to 

```
instead of
```
Hey!

I'm sending you 100 DOTs as a gift! You can go to 
```
Maybe we need to have the entire `giftMessage` string in one line to avoid that?

Reply:
Good catch. Thanks!